docs/en/observability/create-alerts.asciidoc

```diff
@@ -69,6 +69,66 @@ list an alert on the {observability} Alerts page.
 Only alerts generated by rules relating to Logs, Metrics, Uptime, and APM
 can be viewed on the Alerts page.
 
+[discrete]
+[[create-alerts-configure]]
+== Configure alerts
+
+You may want to disable writing to specific {observability} alert indices or disable all alerts and remove the Alerts page altogether.
+You can do this in {kib} settings.
+
+If you are using our https://www.elastic.co/cloud/elasticsearch-service[hosted {ess}] on {ecloud},
+you'll edit the {kib} user settings:
+
+. Select your deployment on the home page, and from your deployment menu go to the *Edit* page.
+. In the *{kib}* section, click *Edit user settings*, and add the desired settings (detailed below).
+. Click *Back*, and then click *Save*. The changes are automatically appended to the `kibana.yml` configuration file for your instance.
+
+If you have a self-managed {stack}, you'll edit the settings in your `kibana.yml` file.
+
+[discrete]
+[[create-alerts-disable-some]]
+=== Disable writing to specific alert indices
+
+To disable writing to specific {observability} alerts-as-data indices
+while continuing to write to others, use `xpack.ruleRegistry.write.disabledRegistrationContexts`.
+
+You can disable writing to alert indices for:
+
+* Logs (`observability.logs`)
+* Metrics (`observability.metrics`)
+* APM (`observability.apm`)
+* Uptime (`observability.uptime`)
+
+NOTE: Disabling writing to uptime alert indices will affect _all_ uptime rule types including monitor status and TLS rule types.
```

Comment:
@colleenmcginnis This is definitely correct and you can keep it as it is! I am just wondering if it is clear to the user that same applies to rest observability rule types as well. Maybe we could have a more generic sentence that follows with a more specific example as you have here. For example `Disabling writing to the indices of an observability solution, will affect all rule types of the corresponding solution. For example disabling writing to uptime alert indices will affect _all_ uptime rule types including monitor status and TLS rule types.`

This is just a suggestion. Feel free to follow it or not.